sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/implementation/RequestTracingConstants.java

```diff
@@ -27,6 +27,11 @@ public enum RequestTracingConstants {
      */
     KUBERNETES_ENVIRONMENT_VARIABLE("KUBERNETES_PORT"),
     
+    /**
+     * Constant for checking for Container App.
```

Comment:
Keep align with other java doc. `Constant for checking for use in Container App.`